pythonFiles/vscode_datascience_helpers/dataframes/vscodeDataFrame.py

```diff
@@ -3,8 +3,34 @@
 import json as _VSCODE_json
 import pandas.io.json as _VSCODE_pd_json
 
+# PyTorch and TensorFlow tensors which can be converted to numpy arrays
+_VSCODE_allowedTensorTypes = ["Tensor", "EagerTensor"]
+
+# Function that converts tensors to DataFrames
+def convertTensorToDataFrame(ten):
```

Comment:
Unless this changed these functions do go into the users python global namespace. So I think that we need the _VSCODE_ prefix here on the function like the other functions. Otherwise it might mess with an existing convertTensorToDataFrame function.

Reply:
Whoops yes thank you! 

I also wanted to ask: I noticed we explicitly `del` some Python variables that we declare in our scripts. Should I be doing that with the variables I declare as well?

Reply:
Yeah, we were doing that. We should do that for this function as well. Wonder if we are missing any? I don't think it's a big deal since most tools hide stuff that starts with _.

Seems like it might be good to automate this somehow? Feels easy to forget / miss. In any case no need to worry about it for this PR, just add the prefix and `del`.